Lib/glyphsLib/builder/glyph.py

```diff
@@ -151,6 +151,43 @@ def to_ufo_glyph(self, ufo_glyph, layer, glyph):  # noqa: C901
         self.to_ufo_glyph_height_and_vertical_origin(ufo_glyph, layer)
 
 
+def to_ufo_openTypeCategories(self, ufo):
+    # See:
+    # * glyphsLib.builder.features._build_gdef
```

Comment:
I think we should factor out the logic to build the GDEF categories from GlyphData from the existing _build_gdef to a new method that both it and this can use, we don't want to duplicate same logic